doc/manual/src/language/index.md

```diff
@@ -434,7 +434,27 @@ This is an incomplete overview of language features, by example.
   </td>
   <td>
 
-   Add all attributes from the given set to the scope (evaluates to `1`)
+   Add all attributes from the given set to the scope (evaluates to `1`).
+
+   **Warning:**
+   In contrast to `inherit`, `with` does not shadow existing variables with the same name!
```

Comment:
```suggestion
   In contrast to regular bindings (created with `=` or the `inherit` shorthand),
   `with` does not always shadow existing variables with the same name!
```

Reply:
Discarding, as I removed the whole section and linked to the existing one instead.